### ISSUE TITLE
Unable to launch app.main module from app module in signed APK, Where it is working fine in debug build

### ISSUE THREAD

**Issue by kumarswamy-repo** [wequick/Small#570]
- [x] 我已阅读并理解 **[贡献指南](https://github.com/wequick/Small/blob/master/贡献指南.md)**，严格遵循其约定。

## 错误报告
unable to launch MainActivity which is in app.main module from app module

##  你做了什么？

I created two modules app and app.main. I have SampleActivity in app module, in which I called as below
Small.setUp(SampleActivity.this, new Small.OnCompleteListener() {
                    @Override
                    public void onComplete() {
                        Small.openUri("main",SampleActivity.this);
                    }
                });

## 你期望的结果是什么？

It should open MainActivity which is in app.main module

## 实际结果是什么？

It is not opening MainActivity. According to me, it is failing at function Small.setUp()  where it is failing to load LaunchableBundles

## Small环境

1.3.0-beta5 version

### Compile-time

```
  gradle-small plugin : 1.3.0-beta5 (maven)
            small aar : 1.3.0-beta5 (maven)
          gradle core : 4.1
       android plugin : 2.3.0
                   OS : Linux 4.10.0-28-generic (amd64)

```

### Bundles

|  type  |       name       |   PP   |          file           |    size    |
|--------|------------------|--------|-------------------------|------------|
|  host  |  app             |        |                         |            |
|  app   |  app.main        |  0x77  |  *_main.so         |  6.5KB   |

### Runtime

```
  Device : Moto G(4)
     SDK : Android 7.0
     ABI : armeabi
```

**Comment by galenlin** [comment]
Has any console logs? It seems that you hadn't declared the bundles in `assets/bundle.json`.

**Comment by galenlin** [comment]
@kingcxsc 是的，参考宿主分身模块。

**Comment by kumarswamy-repo** [comment]
@galenlin 
There are no errors in the console. Successfully generated .so by running ./gradlew buildLib and ./gradlew buildBundle.
I declared bundles in this path app/src/main/assets/build.json.
Let me know if path is wrong.
Every thing is working fine(launching app.main activity from app module). After creating the signedAPK only this is not working.

**Comment by galenlin** [comment]
@kumarswamy-repo It seems that you hadn't sign the `app.*` bundles with the same signature as the host one. 

Please check if you had defined the **release** `signingConfig` in the host module like the sample code provided in `Sample/app/build.gradle`:

```gradle
    // Signing
    signingConfigs {
        release {
            storeFile file('../sign/release.jks')
            storePassword "5mall@ndro!d"
            keyAlias "small"
            keyPassword "5mall@ndro!d"
        }
    }
    buildTypes {
        release {
            signingConfig signingConfigs.release
        }
    }
```

Further more, you can add a breakpoint at `SoBundleLauncher.java`#118:

```java
    if (!parser.verifyAndExtract(bundle, this)) {
        bundle.setEnabled(false);
        return true; // Got it, but disabled
    }
```

to see whether if the verifying is failed.

**Comment by kumarswamy-repo** [comment]
@galenlin 
The issue got resolved with above mentioned suggestion. Thanks for the update.
When I used **generate signed APK** option from **Build** menu in AndroidStudio, I am able to generate signed apk.
When I used ./gradlew assembleRelease command to generate signed apk, I am getting **error: package net.wequick.small does not exist** in app.* modules.
Could you please look into it.

**Comment by galenlin** [comment]
@kumarswamy-repo You need to specify the explicit module path for the gradle command, as `./gradlew :app:assembleRelease`, which the `:app` is your host module path.

**Comment by galenlin** [comment]
You're welcome. 

By the way, I had resolved #505 in #557 which also supports the android gradle plugin 3.0.
